usr/src/man/man1m/dd.1m

```diff
@@ -1,5 +1,6 @@
 '\" te
 .\" Copyright (c) 2014, Joyent, Inc.  All rights Reserved.
+.\" Copyright (c) 2014 by Delphix. All rights reserved.
```

Comment:
I can't seem to mark the correct line here, but the date in the manual page doesn't seem to have been updated.